### PR TITLE
Tasks for a given user

### PR DESCRIPTION
**Pull request by atd**
Show in index tasks whose author or member is the authenticated user

### CODE REVIEW

File: spec/models/task_spec.rb [hunk 1]
```diff
@@ -27,6 +27,58 @@
 
   end
 
+  describe "" do
```
Comment (nogates) [inline spec/models/task_spec.rb]: no title for this group.... ?

**Review by nogates — APPROVED** [review]
awesome!

File: spec/models/task_spec.rb (hunk 1 above)
Comment (atd) [reply spec/models/task_spec.rb]: It was like a little bit noise putting a title there. I can complete it if you feel uncomfortable about it

Comment (nogates) [reply spec/models/task_spec.rb]: maybe, just name it `.scopes`, or something like that :question: 
but happy if you merge it as it is now

**Pull request merged**